### ISSUE TITLE
ModuleNotFoundError: No module named 'network.Noise'

### ISSUE THREAD

**Issue by FlorianPra** [sh1newu/SepMark#2]
It seems that the module Noise is missing in the folder /network.

**Comment by sh1newu** [comment]
Yeah. Please fix this by commenting out the offending line (" from .Noise import Noise " appears in network/DW_EncoderDecoder.py).

**Comment by FlorianPra** [comment]
Thank you for the answer. It fixes the problem.

**Comment by Pikachu1023** [comment]
> Yeah. Please fix this by commenting out the offending line (" from .Noise import Noise " appears in network/DW_EncoderDecoder.py).

Have you reproduced this code successfully

**Comment by Pikachu1023** [comment]
> It seems that the module Noise is missing in the folder /network.

Have you reproduced this code successfully？

**Comment by FlorianPra** [comment]
I just delete the line.

**Comment by Pikachu1023** [comment]
> 我只是删除该行。
Okay, I get it. After this problem is solved, will there be any other problems in the follow-up of the project

**Comment by sh1newu** [comment]
See the README file, please.
